src/plot_utils.py

```diff
@@ -100,20 +97,47 @@ def heatmap(bdb, bql, selectors=None, **kwargs):
     -------
     clustermap: seaborn.clustermap
     """
-    df = bqlu.cursor_to_df(bdb.execute(bql))
+    assert bql is not None or df is not None
+    if bql is not None:
+        df = bqlu.cursor_to_df(bdb.execute(bql))
+    df.fillna(0, inplace=True)
+    return zmatrix(df, **kwargs)
+
+def selected_heatmaps(bdb, selectors, bql=None, df=None, **kwargs):
+    """Plot heatmaps of pairwise matrix, broken up according to selectors.
```

Comment:
Point out that this is a generator of plot objects rather than something that actually shows them; and document that it yields a sequence of triples rather than just one.